sound/soc/sof/ipc4-topology.c

```diff
@@ -427,6 +427,62 @@ static void sof_ipc4_widget_free_comp_pcm(struct snd_sof_widget *swidget)
 	swidget->private = NULL;
 }
 
+static int sof_ipc4_widget_setup_comp_buffer(struct snd_sof_widget *swidget)
+{
```

Comment:
can we reuse sof_ipc4_widget_setup_pcm() and skip just the gtw_cfg bits?

Reply:
> can we reuse sof_ipc4_widget_setup_pcm() and skip just the gtw_cfg bits?

We need to add some assign different gtw_cfg values, not just skip it.
For example,
	ipc4_copier->data.gtw_cfg.node_id = SOF_IPC4_INVALID_NODE_ID;
	ipc4_copier->data.gtw_cfg.config_length = 1;
	ipc4_copier->ipc_config_size = 0;